source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,35 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+You can watch for changes in MongoDB, such as changes to a collection,
```

Comment:
Thanks for adding specificity here. While it's nearly there, I think the paragraph could flow more smoothly. What do you think about reordering the sentences to the following:

-Explain what Change Streams are and what they are and what they can be opened on
-Explain how to open a Change Stream using NodeJS
-Explain how to iterate through the events in a Change Stream using NodeJS
-Explain how to filter events using NodeJS
-Explain how to close Change Streams using NodeJS

Reply:
Hm, it looks like my current paragraph almost follows that structure. One thing I don't agree with is: "-Explain what Change Streams are and what they are and what they can be opened on". I believe we should not explain what a change stream is, that goes out of the scope of this usage example, and is likely too complex for jasmine. We only want to give an opinionated way to use the `watch()` method for a usage example.

That being said, what do you think about the following paragraph:

You can watch for changes in MongoDB by using the watch() method. Calling the ``watch()`` method on a collection, database, or deployment opens a ``Change Stream`` that allows applications to watch for changes to data and react to those changes. After opening a Change Stream, you can iterate through the change events using the next() method on an instance of a Change Stream. ``watch()`` takes an optional pipeline parameter that is an array of stages to filter the change events output. You can stop watching for changes and free resources by closing the Change Stream using the close() method on an instance of a Change Stream.

It follows the following structure:
- Explains what action the ``watch()`` method does ("You can do <some action that the usage-example method does>" is the structure most usage examples follow)
-Explain how to open a Change Stream using NodeJS
-Explain how to iterate through the events in a Change Stream using NodeJS
-Explain how to filter events using NodeJS
-Explain how to close Change Streams using NodeJS

Reply:
> "-Explain what Change Streams are and what they are and what they can be opened on". I believe we should not explain what a change stream is, that goes out of the scope of this usage example, and is likely too complex for jasmine. 

You already explain this, I am just reordering the content you have. If I take your revised content and paraphrase:

"A Change Stream allows applications to watch for changes to data and react to those changes. <Some text that connects change events that to change streams>".

The reason I think this explanation belongs first is because the reader may not understand what the concept is, but you're introducing syntax which might be more suitable for a Usage Example (in which we assume the reader already knows the concept and we're just explaining the syntax).

Reply:
Sorry, I guess this is a Usage Example. However, the sentence structure does not flow smoothly yet.